cms/static/js/factories/manage_users.js

```diff
@@ -1,203 +1,31 @@
-define(['jquery', 'underscore', 'gettext', 'js/views/feedback_prompt'], function($, _, gettext, PromptView) {
+/*
+    Code for editing users and assigning roles within a course team context.
+*/
+define(['underscore', 'gettext', 'js/views/manage_users_and_roles', 'coffee/src/main'],
```

Comment:
What is coffee/src/main needed for? Please add a comment since there is no way to tell from this file.